templates/configure.mk

```diff
@@ -12,6 +12,7 @@
 # Copyright 2020 <contributor name>
 #
 
+BUILD_BITS=64 # for binaries or 32_and_64 for libraries
```

Comment:
And what about libraries? Shouldn't we set BUILD_BITS only if not yet done?

Reply:
The problem with defaults is that they may not hold anymore... that's why I thougt about adding the comment. Is it the wording that bothers you? Should I write "for executables-only packages or 32_and_64 for libraries"?

Reply:
No, it's ok.